Libs/MRML/DisplayableManager/vtkMRMLViewInteractorStyle.h

```diff
@@ -37,43 +40,44 @@ class vtkTimerLog;
 /// Some additional high-level events (such as click and double-click)
 /// are generated here.
 class VTK_MRML_DISPLAYABLEMANAGER_EXPORT vtkMRMLViewInteractorStyle :
-  public vtkInteractorStyle3D
+  public vtkObject
```

Comment:
Why it is not based on an interactor style?

Reply:
Because our `vtkMRMLViewInteractorStyle` are technically observer and not interactor style anymore. For more details, see section `Refactor Interactor Styles` in https://github.com/Slicer/Slicer/issues/7310

For sake of reducing the amount of changes and allowing easy review, I didn't rename our classes. For more details, see section `Renaming Interactor Styles` in https://github.com/Slicer/Slicer/issues/7310.